install-and-configure/install/multi-cluster/secondary-clusters.md

```diff
@@ -12,7 +12,7 @@ See the [Additional resources](/install-and-configure/install/multi-cluster/seco
 
 ## Kubecost global
```

Comment:
I know you didn't modify this, but what does "Kubecost global" mean here? The primary cluster?

Reply:
My interpretation, is that it refers to components of Kubecost that are certainly running. Whereas sections below such as Grafana may not be running in user environments.

Reply:
Would another way to phrase this be "Kubecost Primary Components"? Otherwise, I'm still not sure I understand as a reader.

Reply:
I've renamed the header to "Kubecost". I was hesitant to use the word "Primary Components" because this doc is specifically about secondary clusters. I also think just using the header "Kubecost" may be fitting because other headers just reference the app names (e.g. Prometheus, Grafana, Node-exporter). Let me know what you think!